.github/workflows/deployStatic.yml

```diff
@@ -435,7 +435,8 @@ jobs:
     name: Deploy Database Setup Image
     runs-on: ubuntu-latest
     timeout-minutes: 30
-    if: ${{ github.event.pull_request.merged == true }}
+    # TODO 1/2: Temporarily disable migrations in prod until the postgres 17 upgrade has been completed
```

Comment:
praise: Woah this is neat!

Reply:
Yee, what I should have done the first time, just didnt think of it :)